projects/plugins/jetpack/tests/e2e/lib/pages/wp-admin/recommendations.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * Internal dependencies
+ */
+import WpPage from '../wp-page';
+
+const PERSONAL_CHECKBOX_SELECTOR = '.jp-checkbox-answer__container:nth-child(1) input';
+const BUSINESS_CHECKBOX_SELECTOR = '.jp-checkbox-answer__container:nth-child(2) input';
+const STORE_CHECKBOX_SELECTOR = '.jp-checkbox-answer__container:nth-child(3) input';
+const OTHER_CHECKBOX_SELECTOR = '.jp-checkbox-answer__container:nth-child(4) input';
+
+export default class RecommendationsPage extends WpPage {
+	constructor( page ) {
+		const url = `${ siteUrl }/wp-admin/admin.php?page=jetpack#/recommendations`;
+		super( page, { expectedSelectors: [ '[class^=jp-recommendations-]' ], url } );
+	}
+
+	isUrlInSyncWithStepName( stepName ) {
+		const url = this.page.url();
+		return url.includes( stepName );
+	}
+
+	async areSiteTypeQuestionsVisible() {
+		const siteTypeQuestionsSelector = '.jp-recommendations-question__site-type-checkboxes';
+		return await this.waitForElementToBeVisible( siteTypeQuestionsSelector );
+	}
+
+	async checkPersonalSiteType() {
+		const personalSiteTypeSelector = PERSONAL_CHECKBOX_SELECTOR;
+		return await this.click( personalSiteTypeSelector );
+	}
+
+	async checkOtherSiteType() {
+		const otherSiteTypeSelector = OTHER_CHECKBOX_SELECTOR;
+		return await this.click( otherSiteTypeSelector );
+	}
+
+	async isPersonalSiteTypeChecked() {
+		return await this.isElementVisible( `${ PERSONAL_CHECKBOX_SELECTOR }:checked` );
```

Comment:
I think a better way is to use Playwright's `isChecked` (https://playwright.dev/docs/api/class-page?_highlight=ischecked#pageischeckedselector-options). We don't have a wrapper method for this in `page-actions.js` yet, as we only added what we needed so far. 
I would add in `page-actions.js`
```
async isElementChecked( selector, timeout = this.timeout ) {
		logger.action( `Checking if element '${ selector }' is checked` );
		return await this.page.isChecked( selector, { timeout } );
}
```
and then use is inside pages with `this.isChecked(selector)`

Reply:
This is a great suggestion. I'm definitely adding that new page action.